tests/test_attributes.py

```diff
@@ -882,6 +882,37 @@ class OuterMapAttribute(MapAttribute):
         assert serialized == {'inner_map': {'M': {'foo': {'S': 'bar'}}}}
 
 
+class TestDynamicMapAttribute:
+
+    class CreatedAtTestModel(Model):
+        class CreatedAtMap(DynamicMapAttribute):
```

Comment:
Should we also test with a subclass of `CreatedAtMap` to test one more level of inheritance?